src/crimson/os/futurized_store.h

```diff
@@ -4,9 +4,8 @@
 #pragma once
 
 #include <string>
-#include <unordered_map>
 #include <map>
-#include <typeinfo>
```

Comment:
Ah, I bet that's a residue of the pre-errorator, throwing-free inspection of `seastar::future`. Thanks for catching this!